src/main/java/com/notify/notify/model/request/AddBlacklistNumbersRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.notify.notify.model.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class AddBlacklistNumbersRequest {
```

Comment:
**AddBlacklistNumbersRequest** naming is not makiing sense to me, please remove ADD from name, becuase it is not a method, class name should be noun.